### ISSUE TITLE
Collection Save no longer returns ID of inserted object

### ISSUE THREAD

**Issue by jonmc12** [Urigo/angular-meteor#686]
Problem:
```javascript
$scope.Todos.save(newTodo).then(function(response) {
    // response used to return: [newTodo._id]
    //
    // response now returns: [{
    //     id: undefined,
    //     action: "inserted"
    // }];
}
```

I corrected this locally by updating [angular-meteor-collection.js, line 85](https://github.com/Urigo/angular-meteor/blob/master/modules/angular-meteor-collection.js#L85) from:

```javascript
var createFulfill = _.partial($meteorUtils.fulfill, deferred, null);
```
to:
```javascript
var createFulfill = _.partial($meteorUtils.fulfill, deferred, null, null);
```
I'm not sure if this fix is comprehensive enough for pull request, and probably needs a test added to spec, but has solved the issue for me temporarily.

**Comment by Urigo** [comment]
@jonmc12 would you want to create a pull request with added tests??  
That could be a first step and then I can review it and add notes

**Comment by josemejiasji** [comment]
+1 on this.

When I change the line you suggested, promise returns _id but not the action.

**Comment by jonmc12** [comment]
@Urigo I've submitted a pull request, would welcome any notes

**Comment by thuanmaster** [comment]
+1 @jonmc12  I changed directly in `angular.js` file in .meteor packages and it works as normally.. Many thanks!

**Comment by Urigo** [comment]
Fixed in `1.0.0-rc.10` , thanks to @jonmc12
